src/components/UserManagement/UserAccount/UserAccount.tsx

```diff
@@ -208,4 +227,24 @@ export default class UserAccount extends React.Component<Props, State> {
       profileImageUrl: null,
     });
   }
+
+  onSaveChangesClick = () => {
+    if (this.updateProfileForm.validateContactInfo()) {
+      const customerInput = new UpdateCustomerInput({
+        firstname: this.updateProfileForm.state.updateProfile.firstName,
+        lastname: this.updateProfileForm.state.updateProfile.lastName,
+        is_subscribed: this.updateProfileForm.state.optIn,
+      });
+
+      if (this.updateProfileForm.state.isEmailChanged) {
+        customerInput.email = this.updateProfileForm.state.updateProfile.email;
+        customerInput.password = this.updateProfileForm.state.updateProfile.password;
+      }
+
+      this.context.updateCustomerV2(customerInput).then((value) => {
```

Comment:
We should handle the "Invalid login or password" here if the user writes the wrong password in the field and show the error in the form.